README.md

```diff
@@ -73,19 +73,19 @@ After you add the installation path to the PATH variable, open a new terminal (
   `/home/username/.appzard`
   
 ## FAQ
-  ### Appzard offline url shows a blank screen
-  If you were redirected to the local instance url, yet it has shown a blank screen, please clear your cache and hard reload the page.
+  ### Appzard offline URL shows a blank screen
+  If you were redirected to the local instance URL, yet it has shown a blank screen, please clear your cache and hard reload the page.
   ### Does Appzard offline requires an internet connection anytime?
-  The advantage of appzard offline is to be able to run appzard workspace locally without internet connection. However, some other functions require internet connection to work properly:
-  - Installation: installing appzard requires internet connection to download installation files from the cloud.
-  - Upgrade: upgrading appzard requires internet connection, to check the latest version, and download the updated files.
-  - `appzard doctor`: appzard doctor requires internet connection, to check the latest version of appzard.
+  The advantage of appzard offline is to be able to run appzard workspace locally without an internet connection. However, some other functions require an internet connection to work properly:
+  - Installation: installing Appzard requires an internet connection to download installation files from the cloud.
+  - Upgrade: upgrading appzard requires an internet connection, to check the latest version, and download the updated files.
+  - `appzard doctor`: appzard doctor requires an internet connection, to check the latest version of appzard.
   ### How to stop appzard offline instance
   When starting a new instance you might have got a similar error: `An appzard instance is already running!`
   
-  This error is typically resulted in from two situations:
+  This error typically results in two situations:
```

Comment:
Actually the error is resulted in from the two situations, not the situations are resulted in from the error, so the previous one should be correct.
```suggestion
  This error is typically resulted in from two situations:
```
Or?
```suggestion
  This error is typically caused by two situations:
```

Reply:
okay, thanks